doc-content/enterprise-only/springboot/bus-app-quartz_proc.adoc

```diff
@@ -4,7 +4,7 @@
 If you plan to run your application in a cluster, you must configure the Quartz timer service.
 
 .Prerequisites
-* You have a business application that you created using the http://start.jbpm.org[business application] website and you want to use this application in a cluster.
+* You have a Spring Boot business application and you want to use this application in a cluster.
```

Comment:
Please mentione similar to the below one:
For eg. as below:
You have a Spring Boot business application service file that you created using the maven archetype command. Please refer <<_sect_BA_create_application, Create your business application section>>